R/env.R

```diff
@@ -54,13 +54,13 @@
 #' env(fn)
 #'
 #'
-#' # new_env() creates by default an environment whose parent is the
```

Comment:
I think you're right in general, but in this case I like that `child_` conveys that an environment cannot appear out of nowhere, it always derives from another environment. I'm also considering removing the default argument so as to be always explicit: `child_env(NULL)` (with the understanding that `NULL` refers to the empty environment here). What do you think?